src/hooks/useBasicChartData.ts

```diff
@@ -189,7 +183,7 @@ export default function useBasicChartData(tokens: (Token | null | undefined)[],
     !isKyberDataNotValid && kyberData && chainId
       ? liveDataApi[chainId] + `?ids=${tokenAddresses[0]},${tokenAddresses[1]}`
       : null,
-    fetchKyberDataSWRWithHeader,
+    fetchKyberDataSWR,
```

Comment:
for aggregator api, we must add accept-version header

Reply:
I just merged both functions into one, no need to have multi versions of this

Reply:
no, if domain is price-chart, accept-version will break the api call

Reply:
<img width="846" alt="image" src="https://user-images.githubusercontent.com/11427213/185886771-f30cd7f4-5537-4b20-8580-0674977c80c1.png">

Reply:
only aggregator api accept header `accept-version`

Reply:
ah only for aggregator api, my bad